packages/path_provider/path_provider_android/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 2.2.0
+
+* Adds implementation of getDownloadsDirectory for Android.
 ## NEXT
```

Comment:
Please see https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#updating-a-changelog-that-has-a-next